kubespawner/spawner.py

```diff
@@ -3221,10 +3249,6 @@ def _get_initialized_profile_list(self, profile_list: list):
 
         return profile_list
 
-    # partial set of recognised user_option keys
-    # used for warning about ignoring unrecognised options
-    _user_option_keys = {'profile'}
-
```

Comment:
I inlined this to the logic where it ended up used, in `_options_from_form`. I twasn't correctly named, because it was really `_formdata_keys` or similar rather than valid keys for the parsed `user_options`.